centreon/src/Core/Dashboard/Application/UseCase/AddDashboardToFavorites/AddDashboardToFavorites.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/*
+ * Copyright 2005 - 2023 Centreon (https://www.centreon.com/)
```

Comment:
Beware, it's in your all new files :) 
```suggestion
 * Copyright 2005 - 2024 Centreon (https://www.centreon.com/)
```

Reply:
done ! to valid with this PR : #6241 @adr-mo

Reply:
PR ok and merged